yarn-project/acir-simulator/src/avm/opcodes/accrued_substate.test.ts

```diff
@@ -0,0 +1,84 @@
+import { Fr } from '@aztec/foundation/fields';
+
+import { mock } from 'jest-mock-extended';
+
+import { AvmMachineState } from '../avm_machine_state.js';
+import { initExecutionEnvironment } from '../fixtures/index.js';
+import { HostStorage } from '../journal/host_storage.js';
+import { AvmJournal } from '../journal/journal.js';
+import { EmitNoteHash, EmitNullifier, EmitUnencryptedLog, SendL2ToL1Message } from './accrued_substate.js';
+import { StaticCallStorageAlterError } from './storage.js';
+
+describe('Accrued Substate', () => {
+  let journal: AvmJournal;
+  let machineState: AvmMachineState;
+
+  beforeEach(async () => {
```

Comment:
Why `async` if there are no awaits? (maybe they are nested?)

Another question: can you confirm that Jest handles the `beforeEach(async)` correctly? I was looking around and couldn't find a source of truth explaining if you can expect these to all have been executed before the test starts, etc.

Reply:
Good point -> i think this is lingering from a rebase find and replace. Removing